rotkehlchen/premium/premium.py

```diff
@@ -46,9 +48,24 @@ class RemoteMetadata(NamedTuple):
     data_size: int
 
 
+class UserLimits(TypedDict):
+    limit_of_devices: int
+    pnl_events_limit: int
```

Comment:
add a docstring to this class please explaining what it is

Reply:
+ 1

And explaining what each limit is for in detail